### PR TITLE
Added max length 750 to chat messages

### PR DESCRIPTION
**Pull request by robinkwilson**
Message length is now set at 750. 

If we wanted to make the user experience better, we'd have a pop up over the send button or toast message that doesn't send your message until your characters are under 750 chars.

### CODE REVIEW

File: src/react-components/room/ChatSidebar.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ export function ChatInput(props) {
   return (
     <div className={styles.chatInputContainer}>
       <TextAreaInput
+        maxLength={MAX_MESSAGE_LENGTH}
```
Comment (netpro2k) [inline src/react-components/room/ChatSidebar.js:83]: On the client side I think this check should be sufficient. It will provide the UX to let the user know they have entered too many characters (and we should make sure you cant click send when the form is invalid). I don't think we really need to check in send/receive as well. We ideally should add enforcement on the server side though.

**Review by netpro2k — COMMENTED** [review]
Suggested maybe only checking on the input length rather than checking on send/receive as well. Doing so is fine but it feels a bit redundant. We should at some point add a server side check as well though.

**Comment by keianhzo** [comment]
Same as Dom, this is good for a temp solution but we should enforce this on the server too at some point as it's still possible to repro using the console or we might expose this through the API at some point.

**Review by keianhzo — APPROVED** [review]
Same as Dom, this is good for a temp solution but we should enforce this on the server too at some point as it's still possible to repro using the console or we might expose this through the API at some point.

File: src/react-components/tokens/TokenPageLayout.js [hunk 2]
```diff
@@ -8,13 +8,11 @@ import styleUtils from "../styles/style-utils.scss";
...
 
 export const TokenPageLayout = ({ children }) => (
-  <StorybookAuthContextProvider>
-    <PageContainer className={styles.tokenContainer}>
-      <Column gap="xl" className={classNames(styles.centerPaddingSides, styleUtils.xlPadding)}>
-        {children}
-      </Column>
-    </PageContainer>
-  </StorybookAuthContextProvider>
+  <PageContainer className={styles.tokenContainer}>
```
Comment (netpro2k) [inline src/react-components/tokens/TokenPageLayout.js:11]: This looks unrelated. Don't know the context for this, if this is just an unrelated quick change you wanted to put in with this as well that's all good, just confirming its intentional. I usually try to note this in the PR description so whoever is reviewing knows its on purpose.

File: src/react-components/room/ChatSidebar.js [hunk 3]
```diff
@@ -351,6 +371,7 @@ export function ChatSidebar({ onClose, children, ...rest }) {
       title={<FormattedMessage id="chat-sidebar.title" defaultMessage="Chat" />}
       beforeTitle={<CloseButton onClick={onClose} />}
       contentClassName={styles.content}
+      disableOverflowScroll
```
Comment (netpro2k) [inline src/react-components/room/ChatSidebar.js:374]: Is this actually still needed now? The message list and chat box should both now scroll themselves, so not sure when the container would ever end up overflowing. Not really a problem to explicitly disable it as well, but if it is needed it probably means there is a bug...

**Review by netpro2k — COMMENTED** [review]
Mostly lgtm. Made a few suggestions around using classNames over ternary and integrating ChatLengthWarning more directly into ChatInput but they are pretty minor so totally fine to ignore if you just want to ship this and move on :)

File: src/react-components/room/ChatSidebar.js (hunk 3 above)
Comment (robinkwilson) [reply src/react-components/room/ChatSidebar.js:374]: I kept this inside Sidebar, because in other views like People/Object menus that use the sidebar (not chat), I assume we'll want to enable overflow scroll. Chat is a special case, where the contents manage the scroll themselves.

File: src/react-components/room/ChatSidebarContainer.js [hunk 4]
```diff
@@ -267,6 +272,18 @@ export function ChatSidebarContainer({ scene, canSpawnMessages, presences, occup
         onChange={e => setMessage(e.target.value)}
         placeholder={placeholder}
         value={message}
+        isOverMaxLength={isOverMaxLength}
+        warning={
```
[the remark this answers is not in the document]
Comment (robinkwilson) [reply src/react-components/room/ChatSidebarContainer.js:276]: I find this more readable than adding a `maxLength` prop instead since I only need the calculation in the component. However, I removed the `isOverMaxLength` prop in the `<ChatLengthWarning>` because it already has `messageLength` and `maxLength` everything it needs to calculate it. The extra prop was redundant.
